web/src/components/product/ProductRegistrationPage.tsx

```diff
@@ -87,24 +109,221 @@ const RegisteredProductSection = () => {
   );
 };
 
+type ServerOption = "default" | "custom";
+
+type RegistrationServerProps = {
+  id?: string;
+  value: ServerOption;
+  onChange: (v: ServerOption) => void;
+};
+
+function RegistrationServer({
+  id = "server",
+  value,
+  onChange,
+}: RegistrationServerProps): React.ReactNode {
+  return (
+    <FormGroup fieldId={id} label={SERVER_LABEL}>
+      <Select
+        id={"server"}
+        value={value}
+        label={value === "default" ? SCC_SERVER_LABEL : CUSTOM_SERVER_LABEL}
+        onChange={(v: ServerOption) => onChange(v)}
+      >
+        <SelectList aria-label={_("Server options")}>
+          <SelectOption value="default" description={_("Register to scc.suse.com")}>
```

Comment:
I am not sure how much users are familiar with scc.suse.com...maybe it can be something like `Register to SUSE server`?